### PR TITLE
Convert chat links in badges

### PR DESCRIPTION
**Pull request by jneira**
* Follow up of #7672, it makes them more uniform with the rest of badges
* Also add a relative link to bootstrap subdir

### CODE REVIEW

**Review by Mikolaj — APPROVED** [review]

**Comment by fgaz** [comment]
proposal for the badges: "chat | via IRC" and "chat | via Matrix" which does not imply that they are separate rooms (to me at least)

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
 [![Stackage version](https://www.stackage.org/package/Cabal/badge/lts?label=Stackage)](https://www.stackage.org/package/Cabal)
 [![Artifacts](https://github.com/haskell/cabal/actions/workflows/artifacts.yml/badge.svg?branch=master)](https://github.com/haskell/cabal/actions/workflows/artifacts.yml?query=branch%3Amaster)
 [![Documentation Status](http://readthedocs.org/projects/cabal/badge/?version=latest)](http://cabal.readthedocs.io/en/latest/?badge=latest)
-**[IRC](https://web.libera.chat/#hackage)**
-**[Matrix](https://matrix.to/#/#hackage:libera.chat)**
+[![IRC chat](https://img.shields.io/badge/chat-on%20libera-brightgreen.svg)](https://web.libera.chat/#hackage)
+[![Matrix chat](https://img.shields.io/badge/chat-on%20matrix-brightgreen.svg)](https://matrix.to/#/#hackage:libera.chat)
```
Comment (jneira) [inline README.md:8]:
```suggestion
[![IRC chat](https://img.shields.io/badge/chat-via%20libera-brightgreen.svg)](https://web.libera.chat/#hackage)
[![Matrix chat](https://img.shields.io/badge/chat-via%20matrix-brightgreen.svg)](https://matrix.to/#/#hackage:libera.chat)
```

**Comment by jneira** [comment]
> 
> 
> proposal for the badge text: "chat | via IRC" and "chat | via Matrix" which does not imply that they are separate rooms (to me at least)

agree via is better to make clear they go to the same place, done!

File: README.md (hunk 1 above)
Comment (fgaz) [reply README.md]: "via IRC" maybe? people might not know that libera is an irc server. sorry for the nitpicking, feel free to merge if you disagree

**Review by fgaz — APPROVED** [review]

**Pull request merged**